api/apps/geoprocessing/test/integration/planning-areas/custom-planning-area.repository.e2e-spec.ts

```diff
@@ -0,0 +1,164 @@
+import { INestApplication } from '@nestjs/common';
+import { PromiseType } from 'utility-types';
+import { Repository } from 'typeorm';
+import { getRepositoryToken } from '@nestjs/typeorm';
+import { PlanningArea } from '@marxan/planning-area-repository/planning-area.geo.entity';
+import { CustomPlanningAreaRepository } from '@marxan/planning-area-repository';
+import { bootstrapApplication } from '../../utils';
+
+let fixtures: PromiseType<ReturnType<typeof getFixtures>>;
+let repository: CustomPlanningAreaRepository;
+
+beforeEach(async () => {
+  fixtures = await getFixtures();
+  repository = fixtures.getRepository();
+});
+
+afterEach(async () => {
+  await fixtures.cleanup();
+});
+
+describe(`when entity created`, () => {
+  let id: string;
+  beforeEach(async () => {
+    // given
+    ({ id } = (await repository.saveGeoJson(fixtures.sampleGeoJSON))[0]);
+    fixtures.createdEntities.push(id);
+  });
+
+  it(`should confirm existence of newly created entity `, async () => {
+    // when
+    const hasResult = await repository.has(id);
+
+    // then
+    expect(hasResult).toBe(true);
+  });
+
+  it(`should return a bbox of entity`, async () => {
+    // when
+    const bbox = await repository.getBBox(id);
+
+    // then
+    expect(bbox).toStrictEqual([
+      -30.146484374999996,
+      -40.341796875,
+      38.20365531807149,
+      29.22889003019423,
+    ]);
+  });
+});
+
+it(`should confirm deny existence of not existing entity `, async () => {
```

Comment:
just minor - i think the current test description has some extra leftover words, maybe it can be amended like this?

```suggestion
it(`should deny existence of unknown entity `, async () => {
```

Reply:
yes, sure, fixed :) `confirm deny` 😅